docs/src/gs/create-windows-workload-cluster.md

```diff
@@ -0,0 +1,198 @@
+parameters to specify bare metal shapes instead of using CAPOCI's default virtual
+instance shape. The `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` and `OCI_NODE_PV_TRANSIT_ENCRYPTION`
+parameters disable encryption of data in flight between the bare metal instance and the block storage resources.
+
+> NOTE: The `OCI_NODE_MACHINE_TYPE_OCPUS` must match the OPCU count of the BM shape.
+See the [Compute Shapes][bm-shapes] page to get the OCPU count for the specific shape.
+
+```bash
+OCI_COMPARTMENT_ID=<compartment-id> \
+OCI_CONTROL_PLANE_IMAGE_ID=<linux-custom-image-id> \
+OCI_NODE_IMAGE_ID=<windows-custom-image-id> \
+OCI_SSH_KEY=<ssh-key>  \
+NODE_MACHINE_COUNT=1 \
+OCI_NODE_MACHINE_TYPE=BM.Standard.E4.128 \
+OCI_NODE_MACHINE_TYPE_OCPUS=128 \
+OCI_NODE_PV_TRANSIT_ENCRYPTION=false \
+OCI_CONTROL_PLANE_MACHINE_TYPE_OCPUS=3 \
+OCI_CONTROL_PLANE_MACHINE_TYPE=VM.Standard3.Flex \
+CONTROL_PLANE_MACHINE_COUNT=3 \
+OCI_SHAPE_MEMORY_IN_GBS= \
+KUBERNETES_VERSION=<k8s-version> \
+clusterctl generate cluster <cluster-name> \
+--target-namespace default \
+--flavor windows-calico | kubectl apply -f -
+```
+
+### Access workload cluster Kubeconfig
+
+Execute the following command to list all the workload clusters present:
+
+```bash
+kubectl get clusters -A
+```
+
+Execute the following command to access the kubeconfig of a workload cluster:
+
+```bash
+clusterctl get kubeconfig <cluster-name> -n default > <cluster-name>.kubeconfig
+```
+
+### Install Calico CNI Provider and OCI Cloud Controller Manager
+
+#### Install Calico
+
+The [Calico for Windows][calico-windows] getting started guide should be read for better understand of the CNI on Windows.
+It is recommended to have the following guides handy:
+
+- [Windows Calico Troubleshooting][calico-windows-debug]
+
+##### The steps to follow:
+
+**On the management cluster**
+
+1. Run `kubectl get OCICluster <cluster-name> -o jsonpath='{.spec.controlPlaneEndpoint.host}'` 
+   to get the `KUBERNETES_SERVICE_HOST` and `DNS_NAME_SERVERS` info that will be used in later steps
+
+**On the workload cluster**
+
+1. Download the [v3.24.5 calico release][calico-release]
+1. Uncompress the downloaded file and locate the `calico-vxlan.yaml`, `calico-windows-vxlan.yaml` and `windows-kube-proxy.yaml`
+files in the `manifests` dir
+1. Edit the `calico-vxlan.yaml` and modify the follow variables to allow Calico running on the nodes use VXLAN
+   - `CALICO_IPV4POOL_IPIP` - set to `"Never"`
+   - `CALICO_IPV4POOL_VXLAN` - set to `"Always"`
+   - `CALICO_IPV6POOL_VXLAN` - set to `"Always"`
+1. `kubectl apply -f calico-vxlan.yaml`
+1. Wait for the IPAMConfig to be loaded
+1. `kubectl patch IPAMConfig default --type merge --patch='{"spec": {"strictAffinity": true}}'`
+1. Edit the `calico-windows-vxlan.yaml` and modify the follow variables to allow Calico running on the nodes to talk
+to the Kubernetes control plane
+   - `KUBERNETES_SERVICE_HOST` - the IP address from step 5
```

Comment:
Step 5 does not have IP address related step right?

Reply:
great catch. Updated